### PR TITLE
Refined descriptions

### PR DESCRIPTION
**Pull request by ChristianMayer**

### CODE REVIEW

File: doc/manual/de/config/widgets/index.rst [hunk 1]
```diff
@@ -24,35 +24,35 @@ kann innerhalb der Visu navigiert werden.
...
 +-------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`PushButton <pushbutton/index>`            | Fügt der Visu eine Schaltfläche hinzu, mit der jeweils beim drücken und loslassen ein definierter Wert an den BUS gesendet wird. Z.B. beim Drücken eine 1 und beim Loslassen eine 0. Damit wird es beispielsweise möglich einen Taster zu simulieren um ein Garagentor, Rolläden oder Raffstores zu öffnen und zu schliessen.                               |
+| :doc:`PushButton <pushbutton/index>`            | Fügt der Visu eine Schaltfläche hinzu, mit der beim drücken und loslassen jeweils ein definierter Wert an den BUS gesendet wird. Z.B. beim Drücken eine 1 und beim Loslassen eine 0. Damit ist es z.B. möglich einen Taster zu simulieren um ein Garagentor, Rolläden oder Raffstores zu öffnen und zu schliessen.                               |
```
Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: beim **Drücken** und **Loslassen**

File: doc/manual/de/config/widgets/index.rst [hunk 2]
```diff
@@ -24,35 +24,35 @@ kann innerhalb der Visu navigiert werden.
...
+| :doc:`Refresh <refresh/index>`                  | Mit dem Widget refresh lassen sich die angezeigten Daten der Visu neu abrufen.
 +-------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| :doc:`Reload <reload/index>`                    | Mit dem Widget reload wird der Visu ein Schalter hinzu gefügt, dessen Betätigung die Visu im Browser neu laden lässt??                                                                                                                                                                                                                                      |
+| :doc:`Reload <reload/index>`                    | Mit dem Widget reload wird wie Visu Seite komplett neu geladen.
```
Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: sollte vermutlich "wird die Visu Seite" heißen?

File: doc/manual/de/config/widgets/index.rst [hunk 3]
```diff
@@ -24,35 +24,35 @@ kann innerhalb der Visu navigiert werden.
...
-| :doc:`Trigger <trigger/index>`                  | Fügt der Visu eine Schaltfläche hinzu, mit der genau ein definierter Wert für einen kurzen, sowie ein definierter Wert für einen langen Tastendruck an den BUS gesendet werden kann, z.B. für das Abrufen und Abspeichern von Szenen oder dem Fahren von Rollos. (kurz = stopp, lang = fahren). Die GA für Kurz und Langzeit können unterschiedlich sein.   |
+| :doc:`Trigger <trigger/index>`                  | Fügt der Visu eine Schaltfläche hinzu, mit der genau ein definierter Wert für einen kurzen, sowie optional ein definierter Wert für einen langen Tastendruck an den BUS gesendet werden kann, z.B. für das Abrufen und Abspeichern von Szenen oder dem Fahren von Rollos. (kurz = stopp, lang = fahren). Die GA für Kurz und Langzeit können unterschiedlich sein.   |
```
Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: Vielleicht sollten wir überall das KNX-spezifische **GA** durch das allgemeinere **Adresse** ersetzen?

File: doc/manual/de/config/widgets/index.rst [hunk 4]
```diff
@@ -120,13 +120,13 @@ Sonstige Widgets
...
-| :doc:`Navbar <navbar/index>`                    | Mit dem Widget navbar kann man der gesamten Visu ein Navigationsmenü hinzu fügen. Das Menü lässt sich an einer Seite (oben, unten, link, rechts darstellen. Die Navigation durch die Visu lässt sich damit strukturieren.                                                                                                                                   |
+| :doc:`Navbar <navbar/index>`                    | Mit dem Widget navbar kann der gesamten Visu ein Navigationsmenü hinzu gefügt werde. Das Menü lässt sich an einer Seite (oben, unten, link, rechts) darstellen. Die Navigation durch die Visu lässt sich damit strukturieren.                                                                                                                                   |
```
Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: "gefügt werden"

File: doc/manual/de/config/widgets/index.rst [hunk 5]
```diff
@@ -178,11 +178,11 @@ also z.B.:
 +----------------------------------------------------------+---------------------------------------------------------------------+
 | :doc:`CalendarList <plugins/calendarlist/index>`         | externen Kalendar einbinden                                         |
 +----------------------------------------------------------+---------------------------------------------------------------------+
-| :doc:`Clock <plugins/clock/index>`                       | Widget zum Darstellen der Uhrzeit von einer GA. z.B. dem WireGate   |
```
Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: s.o. GA vs. Adresse

File: doc/manual/de/config/widgets/index.rst [hunk 6]
```diff
@@ -24,35 +24,35 @@ kann innerhalb der Visu navigiert werden.
...
-| :doc:`URL-trigger <urltrigger/index>`           | Das UrlTrigger Widget fügt der Visualisierung eine Schaltfläche hinzu, die im Hintergrund Daten von einer URL abfragt.                                                                                                                                                                                                                                      |
+| :doc:`URL-trigger <urltrigger/index>`           | Das UrlTrigger Widget fügt der Visualisierung eine Schaltfläche hinzu, die im Hintergrund ein URL aufruft um dadurch Aktionen auszuführen, z.B. um ein IFTTT Rezept aufzurufen.                                                                                                                                                                                                                                      |
```
Comment (peuter) [inline doc/manual/de/config/widgets/index.rst]: "eine URL aufruft"

**Comment by peuter** [comment]
One additional note for future changes in the docs inside tables: using anf changing content inside tables in RST-Syntax is not very handy as the vertical "lines" must be in line, e.g. something like this:

```
+-------+--------+
|          |      |
+-------+--------+
```
breaks the table, it must always look like this:

```
+-------+--------+
|          |           |
+-------+--------+
```

as one do not see that from within the github-review page but only by checking out the PR and open the file in an editor (or by generating the docs and check for the generator complaining about malformed tables) I did not see that during reviewing the PR.

So this is a reminder for future changes (and reviews) to check the table formatting.

**Comment by ChristianMayer** [comment]
Do you really mean
```
+-------+--------+
|          |           |
+-------+--------+
```
or should it be
```
+-------+--------+
|       |        |
+-------+--------+
```
?

**Comment by ChristianMayer** [comment]
BTW, is it possible to break lines in table cells to keep the overall line length shorter?
These overlong lines make it hard for me to aling the pipe symbols...

i.e.:
```
+-------+--------+
|       | abc de |
|       | fgh i  |
+-------+--------+
```

**Comment by peuter** [comment]
I mean the second example, somehow on the github website the format is slightly different than in the editor during typing. So maybe it looked different depending where you read the comment.

Unfortunately I found no real way to break lines inside a table (there is a way using blockquotes AFAIR, but you have an unwanted indentation when you use it).

**Pull request merged**